config/routes.rb

```diff
@@ -1,9 +1,11 @@
 Rails.application.routes.draw do
+  get 'home/home'
```

Comment:
This route seems unnecessary.  I'd probably stick with just having the `root` defined to point to this controller / action as you've done below.